pkg/download/blob.go

```diff
@@ -78,10 +78,10 @@ func GetSASBlob(blobURI, blobSas, targetDir string) (string, error) {
 		return "", errors.Wrapf(err, "unable to open storage container: %q", blobURI)
 	}
 
-	// Extract the file name only
-	fileName := getFileName(blobURI)
+	// Extract the llob path after container name
```

Comment:
v nit: spelling of blob

Reply:
I'll address this part of https://msazure.visualstudio.com/One/_workitems/edit/15140524
Need to take this to GA today.